config/conf/application.conf

```diff
@@ -7,40 +7,40 @@
 
 com.microsoft.azure.iotsolutions {
   uiconfig {
-    webservice_port : ${http.port}
-    storageadapter-webservice-url: ${?PCS_STORAGEADAPTER_WEBSERVICE_URL}
-    devicesimulation-webservice-url: ${?PCS_DEVICESIMULATION_WEBSERVICE_URL}
-    telemetry-webservice-url: ${?PCS_TELEMETRY_WEBSERVICE_URL}
-    seed-template: "default"
-    azuremaps-key: ${?PCS_AZUREMAPS_KEY}
+    webservicePort : ${http.port}
+    storageAdapterWebServiceUrl: ${?PCS_STORAGEADAPTER_WEBSERVICE_URL}
+    deviceSimulationWebServiceUrl: ${?PCS_DEVICESIMULATION_WEBSERVICE_URL}
+    telemetryWebServiceUrl: ${?PCS_TELEMETRY_WEBSERVICE_URL}
+    seedTemplate: "default"
+    azureMapsKey: ${?PCS_AZUREMAPS_KEY}
```

Comment:
In c# for lot of these static values I think we have hard coded values, if that is true should we follow the same pattern here?